### DISCUSSION TITLE
[REQUEST] Configuring Sounds for Animations and Custom Animations

### DISCUSSION THREAD

**Discussion by mrwallace888** [CapsAdmin/pac3#918] [Help]
On the visual tree/hierarchy on the left, if you place a sound under an Animation or a Custom Animation, you can configure it to work with it.

For example, you can make it so the sound will play every time the animation plays/loops/restarts upon wearing the outfit.

Is this possible?

And is this already possible through proxies or eventing or something? If so, how can I achieve this?

**Comment by mrwallace888** [comment]
Help?

**Comment by Logzombie** [comment]
I was personally thinking of a new part with a similar function, an experimental animation or state part with more of an emphasis on triggering particular parts or sounds (but not exclusive to) based on a frame timeline, working with standalone or repeating animations. Possibly to work in conjunction with Holdtypes too based on player state. This'll make it so that realistic footsteps isn't the only alternative method to achieve footstep sounds. And for more advanced settings, potentially adding "Is busy/override" tickboxes to automatically override others of the same part (E.G Attack state > Walk state) in case working with holdtypes doesn't become a thing.

I've been thinking about something like this for a while, was going to make a post myself until I saw this. Been brainstorming how PAC3 can acquire a part more emphasised on player-made events, since there are an absolute ton of customisation options as of right now. But making long drawn out events can be very finicky with too much of a heavy reliance on TimerX or delay proxies. Mostly just a thought process on how this could be improved.
